### PR TITLE
[Documentation] Add Information on `cargo install --git`

### LINKED ISSUE #89: [Suggestion] `cargo install --git`
In INSTALL.md, section "Build from source", you list the complete way to install Wthrr from the GitHub repository.  There is, however, also a shorter way using Cargo directly: `cargo install --git <repository URL>`.  See the official Cargo documentation (`cargo help install`) for further information.

Cargo will run the same steps in the background but for the end user, executing only one command with the same effect might be more convenient.

I would like to suggest to add this shorter installation method to INSTALL.md.

### PR DESCRIPTION
**Pull request by kevinmatthes**
This PR fixes #89 by adding a new paragraph to `INSTALL.md`.

### CODE REVIEW

**Comment by tobealive** [comment]
I'd say we can remove the `cargo install --git` command from above and change the `Build from source` section as follows.

---

### Build from source

Another way is to compile the app yourself. Assuming the rust toolchain is installed:

- Use cargo install on the git source

  ```
  cargo install --git  https://github.com/tobealive/wthrr-the-weathercrab.git
  ```

- Clone the repo and build the release version

  ```
  git clone https://github.com/tobealive/wthrr-the-weathercrab.git
  cd wthrr-the-weathercrab
  cargo build --release
  ```

  When the build has finished, you'll find the `wthrr` binary inside the `./target/release` directory.

---

This puts `cargo install --git` as first command to build from source and also splits it from `cargo build` as it is not performing exactly the same in terms of having a ./target/release output.

File: INSTALL.md [hunk 1]
```diff
@@ -66,4 +66,12 @@ cd wthrr-the-weathercrab
...
 
-When the build has finished, you'll find the `wthrr` binary inside the `./target/release` directory
+When the build has finished, you'll find the `wthrr` binary inside the `./target/release` directory.
+
+The same result is achieved by calling Cargo directly using the following
+command.  Cargo will execute the same steps in the background such that only one
+instruction needs to be executed.
+
+```bash
+cargo install --git https://github.com/tobealive/wthrr-the-weathercrab
```
[the remark this answers is not in the document]
Comment (kevinmatthes) [reply INSTALL.md:76]: Which typo do you mean?

[the remark this answers is not in the document]
Comment (kevinmatthes) [reply INSTALL.md:76]: You mean the trailing backtick character in line 55, right?  Or have I introduced any typo?

[the remark this answers is not in the document]
Comment (tobealive) [reply INSTALL.md:76]: Yes, I mean in the current install.md.

The backtick and a missing word "Use cargo to install **the** from source"

[the remark this answers is not in the document]
Comment (kevinmatthes) [reply INSTALL.md]: :wrench: :heavy_check_mark:

**Pull request merged**